src/components/task-plan/teacher-task-plans-listing.cjsx

```diff
@@ -80,6 +80,7 @@ TeacherTaskPlanListing = React.createClass
         isLoaded={-> TeacherTaskPlanStore.isLoaded(courseId)}
         isFailed={-> TeacherTaskPlanStore.isFailed(courseId)}
         render={-> <CourseCalendar plansList={plansList} courseId={courseId}/>}
+        renderStatus={-> <CourseCalendar/>}
```

Comment:
If you just render a course calendar when loading it's unclear if the calendar is loaded but there is nothing to show or if it has not loaded yet. Why not just render `Loading Calendar...`?